src/newWizard/Wizard.tsx

```diff
@@ -38,13 +38,14 @@ import WizardFooter, {InheritedWizardFooterProps, WizardFooterProps} from "./Wiz
  * @param {closable} if true user can close the wizard
  * @param {onClose} callback function to call on close of wizard
  * @param {previousText} custom text for previous button
- * @param {showPreviousButton} if true show previous button on wizard else hide
+ * @param {showNext} if true show next button on wizard else hide
```

Comment:
why showPreviousButton is removed?

Reply:
@girishdudhe 'showPreviousButton' has renamed to 'showPrevious' but comment was there for old as well as for new one. So I have just removed old comment